RHIDP-4807: Third-party plugin installation in RHDH (#733)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/dynamic-plugins/assembly-third-party-plugins.adoc

[id="assembly-third-party-plugins"]
= Third-party plugins in {product}
:context: assembly-third-party-plugins

> You can integrate third-party dynamic plugins in {product}. This flexibility allows you to enhance the platform functionality without modifying its source code and rebuilding. To add third-party dynamic plugins, you can export them as derived package.

Reviewer: ```suggestion
You can integrate third-party dynamic plugins into {product) to enhance its functionality without modifying its source code or rebuilding it. To add these plugins, export them as derived packages.
```

While exporting the plugin package, you must ensure that dependencies are correctly bundled or marked as shared, depending on their relationship to the {product-short} environment.

To integrate a third-party plugin into {product-short}:

> . First, obtain the plugin's source code.

Reviewer: ```suggestion
. Obtain the plugin's source code.
```
